### PR TITLE
Feature events

### PR DESCRIPTION
**Pull request by EthanKvachkoff**
Added an events.py where it sends a message when it joins a server

### CODE REVIEW

File: BirthdayBot/Cogs/Registration.py [hunk 1]
```diff
@@ -144,6 +166,66 @@ def check(msg):
...
+                await self.retryLoop(ctx)
+
+    def updateUserInDB(username, new_birthday, discord_id):
+        with session_scope() as session:
+            user_to_update = (
+                session.query(DiscordUser)
+                .filter(DiscordUser.username == username)
+                .first()
+            )
+            user_to_update.Birthday
```
Comment (dankrzeminski32) [inline BirthdayBot/Cogs/Registration.py:230]: let me fix this real quick

Comment (EthanKvachkoff) [reply BirthdayBot/Cogs/Registration.py:230]: SGTM

**Review by dankrzeminski32 — APPROVED** [review]
Dangerous

**Review by dankrzeminski32 — APPROVED** [review]
Dangerous

**Review by dankrzeminski32 — APPROVED** [review]
Dangerous

**Pull request merged**